docs/README.md

```diff
@@ -1,68 +1,89 @@
+    <a href="https://github.com/Mirascope/mirascope/stargazers" target="_blank"><img src="https://img.shields.io/github/stars/Mirascope/mirascope.svg" alt="Stars"/></a>
+</p>
 
 ---
 
-[![GitHub stars](https://img.shields.io/github/stars/Mirascope/mirascope.svg)](https://github.com/Mirascope/mirascope/stargazers)
-[![Documentation](https://img.shields.io/badge/docs-available-brightgreen)](https://docs.mirascope.io/)
-[![](https://github.com/Mirascope/mirascope/actions/workflows/tests.yml/badge.svg?branch=main)](https://github.com/Mirascope/mirascope/actions/workflows/tests.yml)
-[![GitHub issues](https://img.shields.io/github/issues/Mirascope/mirascope.svg)](https://github.com/Mirascope/mirascope/issues)
-[![Github discussions](https://img.shields.io/github/discussions/Mirascope/mirascope)](https:github.com/Mirascope/mirascope/discussions)
-[![GitHub license](https://img.shields.io/github/license/Mirascope/mirascope.svg)](https://github.com/Mirascope/mirascope/blob/main/LICENSE)
-[![PyPI version](https://img.shields.io/pypi/v/mirascope.svg)](https://pypi.python.org/pypi/mirascope)
-[![PyPI pyversions](https://img.shields.io/pypi/pyversions/mirascope.svg)](https://pypi.python.org/pypi/mirascope)
+**Documentation**: <a href="https://docs.mirascope.io" target="_blank">https://docs.mirascope.io</a>
+
+**Source Code**: <a href="https://github.com/Mirascope/mirascope" target="_blank">https://github.com/Mirascope/mirascope</a>
 
 ---
 
 ## Why use Mirascope?
 
-- **Easy**: Designed for ease of use, Mirascope features an intuitive interface and a minimal learning curve
-- **Intuitive**: Get editor support for prompts, eliminating the need to dig through documentation
-- **Durable**: Mirascope offers versatile and adaptable functionality, allowing seamless integration with custom solutions
-- **Integration**: Leveraging Pydantic, Mirascope offers easy integration with JSON Schema and other tools
-- **Maintainability**: Reduce prompt-related bugs and organize prompts with provided utilities
+**Mirascope** is a library purpose-built for Prompt Engineering on top of <a href="https://pydantic.dev" target="_blank">Pydantic</a>:
 
-## Installation
+* **Intuitive**: Editor support that you expect (e.g. **autocompletion**, **inline errors**)
+* **Peace of Mind**: Pydantic together with our Prompt CLI **eliminate prompt-related bugs**.
+* **Durable**: Seamlessly customize and extend functionality. **Never maintain a fork**.
+* **Integration**: Easily integrate with **JSON Schema** and other tools such as **FastAPI**
+* **Convenience**: Tooling that is **clean**, **elegant**, and **delightful** that **you don't need to maintain**.
+* **Open**: Dedication to building **open-source tools** you can use with **your choice of LLM**.
 
-Install Mirascope and start building with LLMs in minutes.
+## 🚨 Warning: Strong Opinion
```

Comment:
dont even think this is necessary

Reply:
what part of it isn't necessary?

Reply:
the header

Reply:
@wcbudz @brianpark0126 what are your thoughts on this?

I tried it without the header and it made the first section feel way too large, so I added this for some flare and also demarcation.